docs/mdbook/src/android/geojson-guide.md

```diff
@@ -0,0 +1,349 @@
+# Introduction
+
+This guide will teach you how to use [`GeoJsonSource`]("https://maplibre.org/maplibre-native/android/api/-map-libre%20-native%20-android/org.maplibre.android.style.sources/-geo-json-source/index.html") by deep diving into [`GeoJSON`]("https://geojson.org/") file format.
+You will start with fundamentals of how a map renders data internally and why you should prefer [JSON]("https://en.wikipedia.org/wiki/JSON") format in storing geospatial data
+
+# Goals
```

Comment:
Please use proper spacing for Markdown. I.e. have a newline after headings, when a list starts, etc.